test/integration/util/util.go

```diff
@@ -152,7 +145,7 @@ func (m *MinishiftRunner) GetStatus() string {
 }
 
 func (m *MinishiftRunner) CheckStatus(desired string) bool {
-	return m.GetStatus() == desired
+	return strings.Contains(m.GetStatus(), desired)
```

Comment:
This is only about Minishift in the current tests... so name should be `CheckMinishiftStatus` and check for `contains/matches "Minishift: %s"`

Reply:
@gbraad If you see this function is only used for `Does not Exist` when `EnsureDeleted` called and here "Minishift: %s"  is not comparable. If in future we need it some other place then we should pass `desired` state variable as `'minishift:  Running|stopped`

Reply:
maybe a candidate to be removed...